rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -1148,6 +1148,80 @@ void ppu_load_exec(const ppu_exec_object& elf)
 	// Initialize HLE modules
 	ppu_initialize_modules(link);
 
+	// Embedded SPU elf patching
+	for (u32 i = _main->segs[0].addr; i < (_main->segs[0].addr + _main->segs[0].size); i += 4)
```

Comment:
hmm why increments of 4? worth checking the alignment of embedded spu images.

Reply:
Seems to be 32 bit aligned in all the games I tested before writing this, looked at at least 8 games.